pkgs/tools/filesystems/genromfs/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl }:
+
+let
+  version = "0.5.2";
+in
+stdenv.mkDerivation rec {
+  name = "genromfs-${version}";
+
+  src = fetchurl {
+    url = "http://iweb.dl.sourceforge.net/project/romfs/genromfs/${version}/${name}.tar.gz";
+    sha256 = "0q6rpq7cmclmb4ayfyknvzbqysxs4fy8aiahlax1sb2p6k3pzwrh";
+  };
+
+  configurePhase = ''
```

Comment:
It makes more sense to do this in `postPatch`; here you're overriding the entire configuration phase with something that has nothing to do with configuration.